### DISCUSSION TITLE
How do I use osgearth's SDK to get an elevation grayscale image of a rectangular region?

### DISCUSSION THREAD

**Discussion by mirro187** [pelicanmapping/osgearth#1888] [Support Q&A]

**Comment by mirro187** [comment]
~~~~~~~~~~~~~~~~~~~~

**Comment by gwaldron** [comment]
Use a `ContourMapLayer` and either specify your custom stops in the Options, or set a custom TransferFuction. (Note, ignore the set/getGrayscale methods as they are deprecated.)

**Reply by mirro187** [reply]
> Use a `ContourMapLayer` and either specify your custom stops in the Options, or set a custom TransferFuction. (Note, ignore the set/getGrayscale methods as they are deprecated.)

Sorry, I didn't make myself clear. I mean pick up a rectangular area in the scene and use the osgearth SDK to get the elevation data of the rectangular area to generate a grayscale image.
